.golangci.yml

```diff
@@ -36,9 +36,6 @@ linters:
       enable-all: true
       disable:
         - fieldalignment
-      settings:
-        shadow:
-          strict: true
```

Comment:
Hmm, I don't remember where this came from, but it's noisy -- let's use the defaults instead.